src/main/java/magicbees/main/Config.java

```diff
@@ -482,6 +487,11 @@ private void setupThaumcraftItems() {
 
             voidGrafter = new ItemVoidGrafter();
             GameRegistry.registerItem(voidGrafter, voidGrafter.getUnlocalizedName(), CommonProxy.DOMAIN);
+
+            if (Loader.isModLoaded("AWWayofTime")) {
+                BloodHelper.getItems();
+            }
+
```

Comment:
Please move this to the BloodHelper as well as the Items uptop here and use it correctly as an IModHelper (register it in the ModHelperManager)

Reply:
Nah that needs to stay where it is, I did switch it out for a boolean though, but the entire mod handles the items like that so.

Reply:
it has been registered though regardless